__tests__/app.test.js

```diff
@@ -44,12 +44,12 @@ describe("GET /api/topics", () => {
 });
 
 describe("GET /api/articles/:article_id", () => {
-  test("gives status of 200 and responds with the appropriate article", () => {
+  test("gives status of 200 and responds with the appropriate article with comment count", () => {
     return request(app)
       .get("/api/articles/2")
       .expect(200)
       .then(({ body: { chosenArticle } }) => {
-        expect(Object.keys(chosenArticle)).toHaveLength(8);
+        expect(Object.keys(chosenArticle)).toHaveLength(9);
```

Comment:
I understand why you have amended the existing test but actually it's a good test with the right matcher so won't fail if we add keys. I'd be more inclined to write a new test when the api is extended and just test for the new property.